app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -185,7 +189,8 @@ object NavigationItemDestinationsRoutes {
 }
 
 private const val EXTRA_HOME_TAB_ITEM = "extra_home_tab_item"
-private const val EXTRA_USER_ID = "extra_user_id"
+const val EXTRA_USER_ID = "extra_user_id"
+const val EXTRA_INTERNAL_STATUS = "extra_internal_status"
```

Comment:
What does `internal` mean?

Reply:
this is only to make new converastion work, it will be a info later from Kalium, so that we either display a UserProfile for a user that is connected or not connected, as we discussed for now known contacts are connected not known contacts are not

Reply:
will a todo, I added in differente places

Reply:
I guess it's a problem with the naming really.

An `External` user in Wire means the user comes from a different backend _i.e._ has a different `domain` than yours.

I would assume that `Internal` means you both are in the same `domain`.

But this `internal` here is used to flag if it's a contact or not.
However, the connection status can be:
- Not-Connected
- Pending (you are waiting for confirmation from the other user)
- Pending (the other user is waiting for your confirmation)
- Connected
- Blocked

So, internal is not really a good name. I am still not sure what it means.
And I don't really think it should matter.

If you want to get a user profile, you get a user profile. You shouldn't tell Kalium `getContactUserProfile` or `getNotContactUserProfile`.

You should tell Kalium to `getSomeProfile`, and the response would contain the connection status. Kalium already knows stuff, we don't need to bring extra duplicated logic to Reloaded.

Reply:
That is how it is going to work in the future, once we have more info from the Kalium if it is connected or not, we have a discussion that for now, it is okay to assume the known contacts are not connected and public ones from the back-end are not, I need this info to correctly display either the status of the OtherProfileScreen to give the options to send the connnect request from the UI or to open a conversation is it is connected user. So I guess i'll change it to EXTRA_CONNECTED_STATUS for now to avoid confusion for other people that may be reading it